### DISCUSSION TITLE
Created a Basic Report Parser

### DISCUSSION THREAD

**Discussion by thedunston** [intelowlproject/IntelOwl#2463] [Show and tell]
I did some pair-programming with ChatGPT (I'm not good with navigating and traversing complex JSON files or CSS) and created a parser that generates text files from the various analyzers in the report. It creates a basic HTML page that links to each text file:

https://github.com/thedunston/intelOwlParser

It was rough trying to go through the report in JSON and the JSON web view.

**Comment by mlodic** [comment]
hey, thank you for your feedback. We know that JSON is hard. We created the "Visualizer" plugin to help in this: https://intelowlproject.github.io/docs/IntelOwl/usage/?h=visualizer#visualizers. Some default Playbooks have already built visualizations in place. You can create your own ones and have your custom GUI directly in the application itself.

**Reply by thedunston** [reply]
I tried to view it but the page is broken:

https://intelowlproject.github.io/docs/IntelOwl/usage/Contribute.html#how-to-add-a-new-visualizer

I see some very strange output.
![Screenshot 2024-08-17 143607](https://github.com/user-attachments/assets/fecb1291-f097-4d87-bee4-1b878d4a53a7)

**Reply by mlodic** [reply]
oh sorry wrong link: https://intelowlproject.github.io/docs/IntelOwl/usage/#visualizers

**Reply by mlodic** [reply]
and how to create a custom visualizer: https://intelowlproject.github.io/docs/IntelOwl/contribute/#how-to-add-a-new-visualizer
